packages/core-utils/src/route.ts

```diff
@@ -420,7 +432,9 @@ export function makeRouteComparator(
 ): (a: number, b: number) => number {
   return makeMultiCriteriaSort(
     makeTransitOperatorComparator(transitOperators),
-    makeNumericValueComparator(obj => getRouteSortOrderValue(obj.sortOrder)),
+    makeNumericValueComparator(obj =>
+      getOTP1RouteSortOrderValue(obj.sortOrder)
```

Comment:
I like that we are clarifying about OTP1, but could we do it in comments instead of the method name?